egs/wsj/s5/utils/slurm.pl

```diff
@@ -95,6 +95,12 @@ ()
   exit 1;
 }
 
+sub exec_command {
+  my $command = join ' ', @_;
+  # To get the actual exit value, shift right by eight bits.
+  ($_ = qx{$command 2>&1}, $? >> 8);
```

Comment:
From `system()` it's impossible to get the output of the command, but `qx` and backticks are equivalent, so we should be able to use backticks.